templates/devspace.yaml.tpl

```diff
@@ -365,11 +365,6 @@ profiles:
     activation:
       - vars:
           DEVENV_DEV_SKIP_PORTFORWARDING: "true"
```

Comment:
question: do we need to get rid of lines 363-367 too?

Reply:
No, we still need the `skipPortForwarding` devspace profile, which will be triggered by the env : `DEVENV_DEV_SKIP_PORTFORWARDING: "true"`